spec/services/authorisation_spec.rb

```diff
@@ -115,15 +115,31 @@
     subject do
       described_class.new(
         organisation_id: '939eac36-0777-48c2-9c2c-b87c948a9ee0',
-        user_id: '161d1f6a-44f1-4a1a-940d-d1088c439da7'
+        user_id: user_id
       )
     end
+    let(:user_id) { 'default_id' }
 
-    before(:each) do
-      stub_sign_in_with_multiple_organisations
+    context 'user is a member of multiple organisations' do
+      let(:user_id) { '161d1f6a-44f1-4a1a-940d-d1088c439da7' }
+      it 'should find that the user is a member of multiple organisations' do
+        stub_sign_in_with_multiple_organisations(user_id: user_id)
+        expect(subject.many?).to be(true)
+      end
+
+      let(:user_id) { 'another_user_id' }
```

Comment:
There are 2 lets for the same variable in the same context.  Not sure this is behaving the way you expect.

Reply:
Resolved this in 5db5651